onnxruntime/core/graph/contrib_ops/quantization_defs.cc

```diff
@@ -9,15 +9,9 @@
 namespace ONNX_NAMESPACE {
 void RNNShapeInference(InferenceContext& ctx);
 
-void convPoolShapeInference(
-    ONNX_NAMESPACE::InferenceContext& ctx,
-    bool use_dilation, bool require_kernel_shape,
-    int input1Idx,
-    int input2Idx);
-void matmulShapeInference(
-    ONNX_NAMESPACE::InferenceContext& ctx,
-    int input1Idx,
-    int input2Idx);
+void convPoolShapeInference(ONNX_NAMESPACE::InferenceContext& ctx, bool use_dilation, bool require_kernel_shape,
```

Comment:
extra diffs were generated since I ran the clang-format tool on the files touched by this PR. I used the .clang-format file in the repo.